src/templates/pycontw-2020/contents/zh/registration/financial-aid.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}財務補助{% endblock title %}
+
+{% block meta_description %}如何申請 PyCon Taiwan 財務補助，讓我們協助你參與會議。{% endblock meta_description %}
+
+
+{% block content %}
+
+<h1>財務補助</h1>
+
+<h2>方案介紹</h2>
+
+<p><a href="http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays" target="_blank" rel="noopener">Everybody Pays（公平付費）</a>是 PyCon 的核心價值，亦即除特邀來賓外，所有的會議參與者，包括講者與志工，皆需自己付費，方能參加會議。在維持 Everybody Pays 此一核心價值的同時，我們也規劃了財務補助方案，希望協助對會議有興趣，但由於成本而難以參加的朋友們，能夠參與這次會議。我們採取審核制，請有需求的朋友主動與我們聯絡，我們收到所有申請後，會根據預算、申請者狀況等因素，決定是否給予補助，以及其資源多寡。</p>
```

Comment:
We no longer use "Everybody pays". Instead, we use "Everybody contributes". See [PyCon TW 2020 CFP](https://tw.pycon.org/2020/zh-hant/speaking/cfp/).